docs/flow/modin/experimental/pandas.rst

```diff
@@ -12,5 +12,4 @@ Experimental API Reference
 .. autofunction:: read_sql
 .. autofunction:: read_csv_glob
 .. autofunction:: read_pickle_distributed
-.. autoclass:: DataFrame
-   :members: to_pickle_distributed
```

Comment:
There is no need to show `DataFrame` class documentation.

Reply:
The new implementation does not create another target with the `DataFrame` name.